draft-ietf-tls-tls13.md

```diff
@@ -3443,6 +3423,10 @@ ticket_lifetime
   for a shorter period of time than what is stated in the
   ticket_lifetime.
 
+ticket_age_mask
+: A randomly 32-bit value that is used to mask the age of the ticket that the
+  client includes in the "early_data" extension.
```

Comment:
Do you think it's worth mentioning that this value needs to be part of the server's session state? (Either stashed into the ticket or in your stateful session cache.)